rustic-cargo.el

```diff
@@ -428,6 +428,15 @@ If BIN is not nil, create a binary application, otherwise a library."
   (interactive)
   (rustic-run-cargo-command "cargo build"))
 
+;;;###autolod
+(defun rustic-run-shell-command (&optional arg)
+  "Run an arbitrary shell command for the current project.
+Example: use it to provide an environment variable to your application like this `evn MYVAR=1 cargo run' so that it can read it at the runtime.
```

Comment:
evn -> env

Reply:
```suggestion
Example: use it to provide an environment variable to your application like this `env MYVAR=1 cargo run' so that it can read it at the runtime.
```